src/Microsoft.AspNetCore.OData/Builder/ODataConventionModelBuilder.cs

```diff
@@ -39,7 +38,17 @@ private ODataConventionModelBuilder()
         /// the functionality is due to the way assembly resolution is done in AspNet vs AspnetCore.
         /// </remarks>
         public ODataConventionModelBuilder(IServiceProvider provider)
-            : this(provider, isQueryCompositionMode: false)
+            : this(provider, null, isQueryCompositionMode: false)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ODataConventionModelBuilder"/> class.
+        /// </summary>
+        /// <param name="applicationPartManager">The application part manager to use.</param>
+        /// <remarks> This function uses types that are AspNetCore-specific.</remarks>
```

Comment:
I would adjust these remarks to have a similar compatibility format as mentioned above. Also, a `<code>`  sample here of how to use this properly on ASP.NET Core would be amazing.